lib/exercises.rb

```diff
@@ -1,19 +1,41 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n * m) n is the length of the array, m is the max length of string within the array. 
+# Could pull chars method to run first on each element of strings, for O(n) n being the longest string or array.
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 But the time complexity is O(n) if the strings are limited in size (so you can ignore the sort because it's limited in size), or O(n * m log m) where m is the length of the strings, if the string is not limited in size.